projects/plugins/jetpack/modules/publicize.php

```diff
@@ -47,11 +47,6 @@ public function __construct() {
 		if ( $this->in_jetpack ) {
 			Jetpack::enable_module_configurable( __FILE__ );
 
-			// if sharedaddy isn't active, the sharing menu hasn't been added yet.
-			if ( $this->modules->is_active( 'publicize' ) && ! $this->modules->is_active( 'sharedaddy' ) ) {
-				add_action( 'admin_menu', array( &$publicize_ui, 'sharing_menu' ) );
-			}
```

Comment:
Same as above. If we do not need the wp-admin Publicize settings, we would want to deprecate the different methods linked to it, so they can be removed in a few releases.